config/chime_science_run_gpu.yaml

```diff
@@ -110,6 +110,8 @@ updatable_config:
   frb_gain:
     kotekan_update_endpoint: json
     frb_gain_dir: /mnt/frb-archiver/daily_gain_solutions/Latest_FRB
+  UT1_UTC_offset:
+    UT1_UTC_val: 0.00
```

Comment:
Should we make this a recent value, so we have at least a rough estimate when we start the system?    We might also want to have some system in the scheduler to check if the value is old?   e.g. in the case we fully restart the array, and it's still some number of hours before the next update?   Alternatively maybe the TBS should update every 30-60 minutes?

Reply:
Set the value to recent value.